assets/src/edit-story/app/media/local/test/useContextValueProvider.js

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react-hooks';
+
+/**
+ * Internal dependencies
+ */
+import ApiContext from '../../../api/context';
+import useContextValueProvider from '../useContextValueProvider';
+import {
+  GET_MEDIA_RESPONSE_BODY,
+  GET_MEDIA_RESPONSE_HEADER,
+} from '../../../api/test/sampleApiResponses';
+import { ConfigProvider } from '../../../config';
+
+jest.mock('../../useUploadMedia');
```

Comment:
imports should still be listed in the right section, the ordering of jest.mock and import doesn't affect the test.

Reply:
Done.